src/types/ResponseTypes.ts

```diff
@@ -156,12 +152,25 @@ export type TaskStatusFailed = TaskStatusType<StatusState.FAILED, {}>;
 export type TaskStatusRunning = TaskStatusType<StatusState.RUNNING, { estimatedTime?: TaskTime }>;
 
 type TaskAllStatuses =
-	| TaskSatusCompleted
+	| TaskStatusCompleted
 	| TaskStatusPending
 	| TaskStatusFailed
 	| TaskStatusRunning;
 
-type TaskUnknownStatus = Partial<ObjUnionToKeyUnion<TaskAllStatuses>>;
+type TaskUnknownStatus = {
+	startTime?: Date | undefined;
+	endTime?: Date | undefined;
+	taskState?:
+		| StatusState.PENDING
+		| StatusState.RUNNING
+		| StatusState.FAILED
+		| StatusState.COMPLETED
+		| undefined;
+	requestedPrimaries?: number | undefined;
+	simulatedPrimaries?: number | undefined;
+	taskId?: number | undefined;
+	estimatedTime?: TaskTime | undefined;
+}; //Partial<ObjUnionToKeyUnion<TaskAllStatuses>>;
```

Comment:
This comment is important because I don't understand it.

Reply:
You could say I cached type so that it will be more readable in places like this:
![image](https://github.com/yaptide/ui/assets/26445545/a1fcb761-5b39-43f4-be1d-2b7a3f596d6c)
Instead of this
![image](https://github.com/yaptide/ui/assets/26445545/75e18f8b-3b89-4519-8b21-89def8abbaa6)